index.html

```diff
@@ -367,7 +367,8 @@ <h4>Community</h4>
             <li><em>Twitter:</em> <a href="https://twitter.com/openturns_org">@openturns_org</a></li>
             <li><em>StackOverflow:</em> <a href="https://stackoverflow.com/questions/tagged/openturns">[openturns]</a></li>
             <li><em>Publications:</em> <a href="https://arxiv.org/abs/1501.05242">baudin2015</a></li>
-            <li><em>Related software:</em> <a href="https://raven.inl.gov">RAVEN</a>, <a href="https://www.uqlab.com/">UQLab</a>, <a href="https://www.persalys.fr">Persalys</a></li>
+            <li><em>Related software:</em> <a href="https://www.persalys.fr">Persalys</a></li>
+            <li><a href="https://github.com/openturns/www/blob/master/LegalInformation.txt">Legal information and cookie policy</a></li>
```

Comment:
I wouldnt put it in the "Community" column, but rather after the logos

Reply:
Ok. I fixed that.